src/SlateCommandExt.js

```diff
@@ -132,6 +132,13 @@ export class SlateRootWidget extends Widget {
         if (!firefly.originalAppProps) {
             props.menu= fallbackMenu;
         }
+        if (fireflyURL.endsWith('irsaviewer')) {
```

Comment:
What is it about IRSA Viewer that made it necessary to handle things differently?  Is this perhaps a more general problem that could arise in other applications?

Reply:
IRSAViewer sets relative file path for firefly app icon and since jupyter lab is on a different host, it was unable to find that icon file (note this won't be a problem in firefly client that opens in its own browser tab because host will be the same). Also, you can notice at IRSAViewer, icon is positioned differently - overlapping on IRSA's omnipresent banner - but that banner doesn't appear in jupyter lab, so I had to unset that position styling.

Yes, this might be a more general problem esp the relative to absolute path conversion. I didn't have access to RSP or other platforms using it, but if you can test it or tell me how to test, we can make this fix more generic.